### PR TITLE
[DOC-3468] Update Install a delegate with 3P tool custom binaries

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update Install a delegate with 3P tool custom binaries with an ` initScript ` example

For reviewers, a preview is available.

[Install a delegate with third-party tool custom binaries](https://64e542ed748abc3b3340e54d--harness-developer.netlify.app/docs/platform/Delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries#edit-the-delegate-yaml)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/2_Delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md [hunk 1]
```diff
@@ -40,6 +40,17 @@ Before you run the delegate, edit the YAML file to change the following:
 
 :::info note
 For delegate Helm chart deployments, add your third-party tool custom binaries to `initScript` in your `values.yaml` file to run them before delegate installation. The default [values.yaml](https://github.com/harness/delegate-helm-chart/blob/main/harness-delegate-ng/values.yaml) is located in the [delegate-helm-chart](https://github.com/harness/delegate-helm-chart) GitHub repo.
+
+For example, the `values.yaml` entry below installs Kubectl.
+
+```yaml
+# Script to run before delegate installation
+initScript: "
+            curl -L0 https://dl.k8s.io/release/v1.24.3/bin/linux/amd64/kubectl -o kubectl
```
Comment (xingchi-jin) [inline docs/platform/2_Delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:49]: i m not certain about one thing.
if the ARCH is not amd4, i think the url will be different. 
Could you try install a delegate on ubuntu vm and add these commands in init script ?

Comment (xingchi-jin) [reply docs/platform/2_Delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:49]: This is kubectl document, that tells what links to use in which arch.  need to elaborate this here also

Comment (brian-f-harness) [reply docs/platform/2_Delegates/install-delegates/install-a-delegate-with-3-rd-party-tool-custom-binaries.md:49]: @xingchi-jin I added details to the example for arch clarification. We have this install command in other places in the delegate docs and can't cover all arch possibilities.

**Review by pratmit — COMMENTED** [review]
Can we add the 3P link to help users when they have to use which arch such as https://pwittrock.github.io/docs/tasks/tools/install-kubectl/

**Review by pratmit — APPROVED** [review]
LGTM

**Review by aimurphy — APPROVED** [review]

**Pull request merged**